docroot/modules/custom/nvli_custom_search/src/Controller/NvliSearch.php

```diff
@@ -27,5 +27,18 @@ public function search($keyword = NULL) {
     $render['form'] = $this->formBuilder()->getForm('Drupal\nvli_custom_search\Form\CustomSearchForm', $keyword);
     return $render;
   }
+  /**
+   * Search Result Page.
+   *
+   * @return form
+   *   Return Search Form.
+   */
+  public function search_page($resource = NULL) {
```

Comment:
@dishabhadra I would like to understand a bit more on this. Can you schedule a call today for the same.